docs/architecture/architecture.md

```diff
@@ -106,7 +109,38 @@ This transition has been delayed until after the release of Clear Containers 3.0
 due to regressions, as described in [runtime issue 407](https://github.com/clearcontainers/runtime/issues/407).
 Once support for features like hotplug are available in `q35`, the project will
 look to transition to this machine type.
- 
+
+### Assets
+
+Additional resources are required to create each QEMU virtual machine
+instance, specifically a guest kernel and a root filesystem image known as the
+"mini O/S".
+
+The locations of these resources are passed to
+QEMU by the runtime (see [Configuration](configuration)).
+
+#### Guest kernel
+
+The guest kernel passed to the QEMU hypervisor and used to boot the virtual
+machine is highly optimised to provide only those services required by a
+container workload.
+
+#### Mini O/S
+
+The root filesystem image is a highly optimised mini operating system based on
+[Clear Linux](https://clearlinux.org/). It is more commonly
+known as the "mini O/S". This mini O/S is extremely minimal and has a highly optimised boot path.
+
+The only services running in the context of the mini O/S are the init daemon
+(`systemd`) and the [Agent](#agent). The real workload the user wishes to run
+executes within a confined context which uses the container manager image as
+its root filesystem.
+
+For example, when `docker run -ti ubuntu sh` is run, the hypervisor will boot
+the mini-OS image using the guest kernel, launch `cc-agent` which will create
```

Comment:
`systemd` launches `cc-agent`, to be more precise.

Reply:
Fixed.